public/app/features/plugins/sql/ResponseParser.test.ts

```diff
@@ -0,0 +1,25 @@
+import { fill } from 'lodash';
+
+import { DataFrameDTO, FieldType, MutableDataFrame } from '@grafana/data';
+
+import { ResponseParser } from './ResponseParser';
+
+describe('transformMetricFindResponse function', () => {
+  it('should handle big arrays', () => {
+    const responseParser = new ResponseParser();
+    const stringValues = new Array(150000);
```

Comment:
Great idea on adding this test! Do we need to import fill from `lodash` here? I believe we can accomplish the same thing with the native `fill()` function -> `new Array(150_000).fill('a');`

Reply:
TIL 😱